packages/app-provider/package.json

```diff
@@ -14,7 +14,7 @@
   "peerDependencies": {
     "apollo-client": "^2.6.3",
     "graphql": "^14.0.0",
-    "react": "^16.3.0",
+    "react": "^16.8.6",
     "react-apollo": "^2.5.0",
     "react-dom": "^16.3.0"
```

Comment:
@zouxuoz I think you also need to updated react-dom.

Reply:
@vorobeez fixed